src/component/settings/SettingsStart.jsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable import/no-unresolved */
+import React, { useContext } from 'react';
+import {
+  Paper, Box, Typography, TextField, Button, Icon,
+} from '@material-ui/core';
+
+import SpacingDesign from '../context/design/SpacingDesign';
+import LogStatus from '../context/auth/LogStatus';
+
+const SettingsStart = () => {
+  const [logStatus] = useContext(LogStatus);
+
+  return (
+    <Paper elevation={3} style={SpacingDesign.padding(3)}>
+      <Typography variant="h5" align="center">
+        Settings
+      </Typography>
+      <TextField
+        fullWidth
+        label="Username"
+        variant="filled"
+        defaultValue={logStatus && logStatus.username}
+        style={SpacingDesign.marginy(1.5)}
+      />
+      <Box display="flex" flexWrap="wrap" justifyContent="space-between" style={SpacingDesign.marginy(1.5)}>
+        <TextField
+          label="First Name"
+          variant="filled"
+          defaultValue={logStatus && logStatus.firstName}
+          style={{
+            flexGrow: 1, flexBasis: '45%', minWidth: '10em', ...SpacingDesign.margin(1),
+          }}
+        />
+        <TextField
+          label="Last Name"
+          variant="filled"
+          defaultValue={logStatus && logStatus.lastName}
+          style={{
+            flexGrow: 1, flexBasis: '45%', minWidth: '10em', ...SpacingDesign.margin(1),
+          }}
+        />
```

Comment:
any way to modularize this/make these into higher-order components?

Reply:
I completely agree. Planning on making it much more modular w/ a new page
Thinking of something similar to this.

![image](https://user-images.githubusercontent.com/47019696/125492767-07956a61-b8ea-4dc3-985a-456d62631bd4.png)

 this comment applies to the other comment as well